webhook_server/libs/handlers/runner_handler.py

```
@@ -435,13 +431,13 @@ async def run_conventional_title_check(self, pull_request: PullRequest) -> None:
         if await self.check_run_handler.is_check_run_in_progress(check_run=CONVENTIONAL_TITLE_STR):
             self.logger.info(f"{self.log_prefix} Check run is in progress, re-running {CONVENTIONAL_TITLE_STR}.")
 
-        await self.check_run_handler.set_conventional_title_in_progress()
+        await self.check_run_handler.set_check_in_progress(name=CONVENTIONAL_TITLE_STR)
         allowed_names = [name.strip() for name in self.github_webhook.conventional_title.split(",") if name.strip()]
         title = pull_request.title
 
         self.logger.debug(f"{self.log_prefix} Conventional title check for title: {title}, allowed: {allowed_names}")
         if any([re.match(rf"^{re.escape(_name)}(\([^)]+\))?!?: .+", title) for _name in allowed_names]):
```

Ruff (0.14.10)

Unnecessary list comprehension

Remove unnecessary comprehension

(C419)